### PR TITLE
storybook-static fixes

### PR DESCRIPTION
**Pull request by hokei**
[Project Reno Jira Ticket](http://jira.nypl.org/browse/RENO-XXX)

[ResearchNow Jira Ticket](http://jira.nypl.org/browse/SFR-XXX)

## **This PR does the following:**
- Fixes so that Storybook will build

### Front End Review:
- [ ] View [the example in Storybook](https://reno-XXX-nypl.pantheonsite.io/themes/custom/nypl_emulsify/pattern-lab/public)
- [ ] Check against the [Metronome documentation](http://themetronome.co/components/xxx/?fresh=true)

### CODE REVIEW

File: src/react-components/webpack.prod.js [hunk 1]
```diff
@@ -10,7 +10,8 @@ module.exports = {
   devtool: "source-map",
   entry: {
     app: [
-      "./src/index.ts"
+      "./src/index.ts", 
+      // "./src/styles.scss"
```
Comment (hokei) [inline src/react-components/webpack.prod.js:14]: so this interferes with importing design system in the consumer app.  I was able to build it without, but ... I'm kind of confused about it

**Comment by hokei** [comment]
ugh this is really frustrating.  The package-lock problem is causing it not to build on travis, but if I revert the package-locks so it can build, it will throw the "cannot find icons" error when trying to `npm link`.  
There's an underlying problem here - perhaps with dependencies?

File: src/react-components/webpack.prod.js (hunk 1 above)
Comment (EdwinGuzman) [reply src/react-components/webpack.prod.js:14]: From this and the removal of `import './styles.scss';`, this means that the scss is not available anymore for the consumer app, right?

**Comment by hokei** [comment]
ugh travis

File: src/react-components/webpack.prod.js (hunk 1 above)
Comment (hokei) [reply src/react-components/webpack.prod.js:14]: because it's being copied over to lib and dist, and then imported directly, it seems to still work.

**Pull request closed without merging**